cmd/invites_approve.go

```diff
@@ -53,7 +53,10 @@ func invitesApprove(ctx *cli.Context) error {
 		return errs.NewExitError("Invite not found.")
 	}
 
-	err = client.OrgInvites.Approve(context.Background(), *targetInvite, progress)
+	s, p := spinner(fmt.Sprintf("Reviewing invite for %s", email))
```

Comment:
I'm thinking the text here should say "Attempting to approve invite for %s" or similar?